packages/expo/README.md

```diff
@@ -35,78 +35,105 @@ Clerk is the easiest way to add authentication and user management to your Expo
+import { StyleSheet, Text, View } from 'react-native';
+import { ClerkProvider } from '@clerk/clerk-expo';
 
 export default function App() {
   return (
-    <ClerkProvider frontendApi={frontendApi}>
-      <Main />
+    <ClerkProvider publishableKey={'your-publishable-key'}>
+      <View style={styles.container}>
+        <Text>Open up App.js to start working on your app!</Text>
+        <StatusBar style='auto' />
+      </View>
     </ClerkProvider>
   );
 }
 
-const Main = withClerk(({ clerk }) => {
-  const { user } = useUser({ withAssertions: true });
-  const handleSignIn = () => clerk.openSignIn();
-  const handleSignOut = () => clerk.signOut();
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+});
+```
 
+A token cache is required to work with Clerk and Expo. This is entirely up to you how you handle the token cache - in this example we're going to use the `expo-secure-store` library. First, install it by running
+
+```
+npm i expo-secure-store
+```
+
+and then add the tokenCache to your entry file, as shown here:
+
+```diff
+import { StatusBar } from "expo-status-bar";
+import { StyleSheet, Text, View } from "react-native";
+import { ClerkProvider } from "@clerk/clerk-expo";
++ import * as SecureStore from "expo-secure-store";
+
++ const tokenCache = {
++  getToken(key) {
++    return SecureStore.getItemAsync(key);
++  },
++  saveToken(key, value) {
++    return SecureStore.setItemAsync(key, value);
++  },
++};
+
+export default function App() {
   return (
-    <View style={styles.container}>
-      {user ? (
-        <>
-          <Text>You are signed in</Text>
-          <TouchableOpacity
-            style={styles.button}
-            onPress={handleSignOut}
-          >
-            <Text>Sign out</Text>
-          </TouchableOpacity>
-        </>
-      ) : (
-        <TouchableOpacity
-          style={styles.button}
-          onPress={handleSignIn}
-        >
-          <Text>Sign in</Text>
-        </TouchableOpacity>
-      )}
-    </View>
+    <ClerkProvider
+      publishableKey={"your-publishable-key"}
++     tokenCache={tokenCache}
+    >
+      <View style={styles.container}>
+        <Text>Open up App.js to start working on your app!</Text>
+        <StatusBar style="auto" />
+      </View>
+    </ClerkProvider>
   );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
+    alignItems: "center",
+    justifyContent: "center",
+  },
 });
+
 ```
 
-_For further details and examples, please refer to our [Documentation](https://clerk.dev/docs?utm_source=github&utm_medium=clerk_react?utm_source=github&utm_medium=clerk_expo)._
+_The section above covers the basic setup. For further details and examples, please refer to our [Clerk Expo Documentation](https://clerk.dev/docs?utm_source=github&utm_medium=clerk_react?utm_source=github&utm_medium=clerk_expo)._
```

Comment:
❓  the url `https://clerk.dev/docs?utm_source=github&utm_medium=clerk_react?utm_source=github&utm_medium=clerk_expo` contains twice the `?` and the `utm_*` params. Shouldn't it be `https://clerk.dev/docs?utm_source=github&utm_medium=clerk_expo` ?

Reply:
Thanks for spotting that :) Fixed!